CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ FetchContent_Declare(imgui         EXCLUDE_FROM_ALL URL https://github.com/ocorn
 # Server dependencies
 FetchContent_Declare(monado
     GIT_REPOSITORY   https://gitlab.freedesktop.org/monado/monado
-    GIT_TAG          598080453545c6bf313829e5780ffb7dde9b79dc
+    GIT_TAG          v24.0.0
```

Comment:
I think it is a bit better to use sha even for a tagged version, cmake won't have to access the remote if it already has the version

Reply:
good call, thanks.